i18n/en/docusaurus-plugin-content-docs/current/guides/examples/entities.mdx

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 5
+---
+
+# Entities
+
+`entities` layer in Feature-Sliced Design encapsulates core business logic, similar to entities in backend applications. However, its use in frontend applications is not always necessary and should be approached with caution to avoid overcomplicating the codebase.
+
+## When to Use the `entities` Layer
+
+Most frontend applications function as "thin clients" with minimal business logic, relying on the backend for data processing. In such cases, `entities` layer can often be omitted in favor of `shared/api`. Use `entities` layer only when the following conditions are met:
+
+- The application handles **client-only data structures** with significant **client-only business logic**.
+- The entity is **heavily reused** across multiple parts of the application.
+
+If you end it a situation where you can't create an entity, but you have a significant amount of reused client-only business logic, you can do a hybrid approach:
+
+- Keep all types/DTOs in `shared/api` without creating a new entity.
+- Use `shared/api` for data fetching operations like create, read, update, delete, filter, etc.
+- Use `entities/<name>/model` for client-only business logic.
+
+### Examples of Valid Use Cases
+1. **Client-Only Entities**: If the frontend manages an entity that does not exist on the backend and involves complex business logic, consider placing it in `entities` layer. This is appropriate only if the entity is used extensively across the application.
```

Comment:
issue: I think we're often using the term "entity" without establishing first what an entity is, even though that's what the whole section is about

suggestion: I think DDD has run up against the same issue before, so we might be able to look to them for inspiration for a defintion of "entity". This question on Stack Overflow, for example, provides some insight, but will still probably need more explanation: https://stackoverflow.com/questions/57367017/the-meaning-of-entity-in-domain-driven-design

Reply:
question: Regarding this part:

> This is appropriate only if the entity is used extensively across the application

I'm struggling to come up with an example of an entity that would have complex business logic, but wouldn't be worthy of being put into the entities layer. Do you have some examples?

Reply:
I will probably need some help with defining "Entity", as for now we only describe it by its properties, and it is challenging to make a good formal description :thinking: 

And tried to specify "heavy" use with https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)